core/lib/spree/testing_support/order_walkthrough.rb

```diff
@@ -3,11 +3,11 @@
 module Spree
   module TestingSupport
     class OrderWalkthrough
-      def self.up_to(state)
-        new.up_to(state)
+      def self.up_to(state, user = nil)
+        new.up_to(state, user)
       end
 
-      def up_to(state)
+      def up_to(state, user = nil)
```

Comment:
ditto

Reply:
I've extracted that commit into its own PR https://github.com/solidusio/solidus/pull/4292